Riot/Modules/Room/RoomViewController.m

```diff
@@ -1338,6 +1348,17 @@ - (void)setScrollToBottomHidden:(BOOL)scrollToBottomHidden
         _scrollToBottomHidden = scrollToBottomHidden;
     }
     
+    if (!_scrollToBottomHidden && [self.roomDataSource isKindOfClass:RoomDataSource.class])
+    {
+        RoomDataSource *roomDataSource = (RoomDataSource *) self.roomDataSource;
+        if (roomDataSource.currentTypingUsers && !roomDataSource.currentTypingUsers.count)
+        {
+            [roomDataSource resetTypingNotification];
+            NSInteger count = [roomDataSource tableView:self.bubblesTableView numberOfRowsInSection:0];
```

Comment:
It may lead to problems to call `UITableView` data source methods on our own. Could you try to use `[self.bubblesTableView numberOfRowsInSection:0];` instead and see that also works?

And also the result of that method doesn't have to be equal to actual number of rows in the table, which may lead to invalid range exceptions.